editor/plugins/cast_2d_editor_plugin.cpp

```diff
@@ -78,7 +78,7 @@ bool Cast2DEditor::forward_canvas_gui_input(const Ref<InputEvent> &p_event) {
 			}
 		} else if (pressed) {
 			EditorUndoRedoManager *undo_redo = EditorUndoRedoManager::get_singleton();
-			undo_redo->create_action(TTR("Set target_position"));
+			undo_redo->create_action(vformat(TTR("Set %s"), "target_position"));
```

Comment:
Don't we write undoredo messages in natural language normally? If so this could be "Set Target Position".

Reply:
The generic undoredo message produced by the Inspector uses snake case property names with `Set %s` though.

Reply:
I'm fine either way, I haven't checked which one is the most common but we should follow that :)

Reply:
In this case, it's modifying `target_position` via a gizmo handle, so both makes sense. But maybe "Set Target Position" is better since the user is not directly setting that property.